pkg/services/object/metrics.go

```diff
@@ -161,8 +171,18 @@ func (s putStreamMetric) Send(req *object.PutRequest) error {
 func (s putStreamMetric) CloseAndRecv() (*object.PutResponse, error) {
 	res, err := s.stream.CloseAndRecv()
 
-	s.metrics.IncPutReqCounter(err == nil)
+	s.metrics.IncPutReqCounter(s.cid, err == nil)
 	s.metrics.AddPutReqDuration(time.Since(s.start))
 
 	return res, err
 }
+
+func getCIDString(cnrV2 *refs.ContainerID) string {
+	var cnr cidSDK.ID
```

Comment:
we already have **v2** struct here, it is quite often operation, mb just `base58.Encode` with comment here?

Reply:
Fixed